wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
And you yourself made the c++ function to only report first argument "param" on error:

catch(...){
 		str2buf(format("parameter is invalid: %s", param), Output, n);
 	}